src/App.tsx

```diff
@@ -17,18 +61,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                search={search}
+                selectMethod={setFilterSelect}
+                searchWord={setSearch}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                todos.length === 0
+                  ? <Loader />
```

Comment:
Currently the Loader is shown if there are no todos. It would be better to have a separate state to track whether loading is in progress or not.